README.md

```diff
@@ -1,6 +1,9 @@
 ![Lint and Test Charts](https://github.com/sonatype/helm3-charts/workflows/Lint%20and%20Test%20Charts/badge.svg)
```

Comment:
we'll need to remove nexus-repository-manager from this badge.svg as well

Reply:
@bobotimi Is this something you need me to do? And, if so, how? haha